addons/stock_account/models/product_category.py

```diff
@@ -0,0 +1,62 @@
+# -*- coding: utf-8 -*-
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import fields, models
+
+
+class ProductCategory(models.Model):
+    _inherit = 'product.category'
+
+    property_valuation = fields.Selection([
+        ('manual_periodic', 'Periodic (manual)'),
+        ('real_time', 'Perpetual (automated)')],
+        string='Inventory Valuation',
+        required=True, copy=True,
```

Comment:
@mdi-odoo,
Yes you are right default copy=True is only for normal field and for other relational field it is False
but here "property_valuation" is property field so that’s why i have to set copy=True.